src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
This is an unrelated component which has already been merged.

Reply:
Also deleted the extra button file that I created for the "Parse button" and reused the "ButtonComponent" that is already present in the repo for the "Parse Button".